packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -292,6 +292,7 @@ export async function getDevMiniflareOptions(config: {
 			],
 			bindings: {
 				CONFIG: assetsConfig,
+				__VITE_HEADERS__: JSON.stringify(viteDevServer.config.server.headers),
```

Comment:
Is this guaranteed to be defined

Reply:
Yep, it gets normalised to an empty object.